examples/how-tos/review-tool-calls.ipynb

```diff
@@ -0,0 +1,762 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import * as tslab from \"tslab\";\n",
+    "\n",
+    "const drawableGraph = graph.getGraph();\n",
+    "const image = await drawableGraph.drawMermaidPng();\n",
+    "const arrayBuffer = await image.arrayBuffer();\n",
+    "\n",
+    "await tslab.display.png(new Uint8Array(arrayBuffer));"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example with no review\n",
+    "\n",
+    "Let's look at an example when no review is required (because no tools are called)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 167,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "================================ human Message (1) =================================\n",
+      "hi!\n",
+      "================================ ai Message (1) =================================\n",
+      "Hello! Welcome. How can I assist you today? Is there anything specific you'd like to know or any information you need help with?\n"
+     ]
+    }
+   ],
+   "source": [
+    "import { HumanMessage } from \"@langchain/core/messages\";\n",
+    "// Input\n",
+    "let inputs = new HumanMessage(\"hi!\");\n",
+    "\n",
+    "// Thread\n",
+    "let config = { configurable: { thread_id: \"1\" }, streamMode: \"values\" as const };\n",
```

Comment:
I think if you do `let config: RunnableConfig = {}` you can avoid the `as const`

Reply:
I was getting an error when I did this: `let config: RunnableConfig = { configurable: { thread_id: "1" }, streamMode: "values" };` lmk what I am doing wrong

Reply:
Oh yeah never mind I know why - it uses an extension of `RunnableConfig`